trestlebot/cli/options/create.py

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Red Hat, Inc.
+
+"""
+Module for common create commands
+"""
+
+import functools
+from typing import Any, Callable, Dict, Sequence, TypeVar
+
+import click
+
+
+F = TypeVar("F", bound=Callable[..., Any])
+
+
+def common_create_options(f: F) -> F:
+    """
+    Configuring common create options decorator for SSP and CD command
+    """
+
+    @click.option(
+        "--profile-name",
+        prompt="Name of trestle workspace to include",
```

Comment:
@hbraswelrh could this prompt be updated to mention profile?  Maybe something like what you have in the help text `Name of profile in trestle workspace to include`?